gen/dcos-config.yaml

```diff
@@ -798,9 +798,8 @@ package:
 {% case "static" %}
   - path: /etc_master/dns_config_master
     content: |
-      MASTER_SOURCE=exhibitor_uri
-      EXHIBITOR_URI=http://127.0.0.1:8181/exhibitor/v1/cluster/status
-      EXHIBITOR_ADDRESS=127.0.0.1
+      MASTER_SOURCE=master_list
+      RESOLVERS={{ resolvers_str }}
```

Comment:
Nice to see that.

Can you link to the corresponding JIRA ticket here as a reply to this comment? At least for posterity.

Reply:
I would prefer that this change was made by the networking team since I still don't know that this does not break anything. It is present in this diff for the sole purpose of passing the tests.

Reply:
Still it would be nice if you could link the JIRA ticket here in this comment stream so that the related discussion in JIRA can be easily discovered by reviewers of your pull request!  :)

Reply:
https://jira.mesosphere.com/browse/DCOS-18703

Reply:
Hi @rdeliallisi!

Could you please remove `RESOLVERS` from this config? It seems like it's already configured in `/opt/mesosphere/etc/dns_config`.

Please check that this patch will work for dcos-enterprice. As I know bootstrap also [uses](https://github.com/mesosphere/dcos-enterprise/blob/master/packages/bootstrap/extra/dcos_internal_utils/bootstrap.py#L1529-L1553) this configuration on agent nodes.

Also I can make it by myself as part of [DCOS-18703](https://jira.mesosphere.com/browse/DCOS-18703) if you prefer.

Reply:
@urbanserj @rdeliallisi's internship at Mesosphere has ended but I'm happy to take his work further. I'll remove the `RESOLVERS` config and check that the upstream bump passes

Reply:
Not all units that source the `dns_config_master` file also source `dns_config`. For example 
https://github.com/dcos/dcos/blob/e61cfc2038146a58614a58989ff68f64dbdc8038/packages/dcos-net/extra/dcos-dns-watchdog.service
https://github.com/dcos/dcos/blob/e61cfc2038146a58614a58989ff68f64dbdc8038/packages/dcos-net/extra/dcos-l4lb-watchdog.servicehttps://github.com/dcos/dcos/blob/e61cfc2038146a58614a58989ff68f64dbdc8038/packages/dcos-net/extra/dcos-overlay-watchdog.service

I suggest we keep the `RESOLVERS=` variable in `dns_config_master` and address that refactoring in a separate PR.